pkg/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Microsoft.ML.TimeSeries contains ML.NET Time Series prediction algorithms. Uses Intel Mkl.</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+	<Content Include="$(PackagesDir)\mlnetmkldeps\$(MlNetMklDepsPackageVersion)\LICENSE.txt" Pack="true" PackagePath=".\NOTICE.txt" /> 
```

Comment:
We should think about making a single MKL nuget package and then having TimeSeries and HalLearners both reference it. As this is today, if someone tried using both TimeSeries and HalLearners, the MklImports asset would collide.

Also, if there are bugs/changes in MKL that we need to take, now we need to service it in 2 separate nuget packages.

Reply:
Agree with Eric here.  We should be making people download this binary twice and relying on the build to de-dup it.

Reply:
I also agree and we spoke offline on this. I have opened a issue for this [https://github.com/dotnet/machinelearning/issues/1092](https://github.com/dotnet/machinelearning/issues/1092)

The challenge here is we know from past experience SymSGD breaks if it tries to consume MKL binaries in the form of a package. Hence we were shipping MKL binaries together with HALLearner package. 

---
In reply to: [221367285](https://github.com/dotnet/machinelearning/pull/977#discussion_r221367285) [](ancestors = 221367285)